### ISSUE TITLE
Learning-Curve

### ISSUE THREAD

**Issue by SonuKumari1690** [NVIDIA/NeMo#3378]
HI Team,
I need to draw learning curve for the trained Biomegatron model. I have trained  the model as per the tutorial https://colab.research.google.com/github/NVIDIA/NeMo/blob/main/tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb#scrollTo=DQhsamclRtxJ

Now when I am trying to get the learning-curve which is the plot for each training loss and validation loss as per the number of epochs as sklearn library providing attribute as from sklearn.model_selection import learning_curve

![image](https://user-images.githubusercontent.com/45452951/147752362-c74ad670-a798-4125-9dc1-60056fb06fc7.png)

**Comment by SonuKumari1690** [comment]
@okuchaiev : Waiting for your kind response

**Comment by VahidooX** [comment]
We already have this feature. These curves are already stored in the folder specified by exp_dir and tensorboard can read and plot them for you.

https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/text_classification/conf/text_classification_config.yaml#L117
